docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -44,6 +48,7 @@ The date must be a hyphen-separated ISO8601 date in the format of:
  * a two digit day.
 
 For example:
+
```

Comment:
IIRC, ReST does not require bullet points to be indented, so it thinks the list is _also_ blockquote'd. The newline indicates that it's a separate paragraph, so its indent is no longer coupled to the first line, and thus within its paragraph, it is consistent and not considered indented any extra.